#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_icmp-manpagefix.dpatch by William Vera <billy@billy.com.mx>
##
## DP: Character cleanup for UTF-8 compatibility.

@DPATCH@
diff -urNad nemesis-1.4~/man/nemesis-icmp.1 nemesis-1.4/man/nemesis-icmp.1
--- nemesis-1.4~/man/nemesis-icmp.1	2003-10-31 15:29:36.000000000 -0600
+++ nemesis-1.4/man/nemesis-icmp.1	2008-07-02 05:43:27.000000000 -0500
@@ -99,10 +99,10 @@
 This will case nemesis-icmp to use the specified
 .I payload-file
 as the payload when injecting ICMP packets.  For packets injected using the
-raw interface (where -d is not used), the maximum payload size is
+raw interface (where \-d is not used), the maximum payload size is
 65387 bytes.  For packets injected using the link layer interface
-(where -d IS used), the maximum payload size is 1352 bytes.  Payloads can also
-be read from stdin by specifying '-P -' instead of a payload file.
+(where \-d IS used), the maximum payload size is 1352 bytes.  Payloads can also
+be read from stdin by specifying '\-P \-' instead of a payload file.
 
 Windows systems are limited to a maximum payload size of 1352 bytes for ICMP 
 packets.
@@ -160,7 +160,7 @@
 Specify the
 .I original-IP-fragmentation-options
 within an ICMP unreachable, redirect or time exceeded packet.  For more 
-information reference the '-F' command line switch.
+information reference the '\-F' command line switch.
 .IP "-j original-IP-TOS"
 Specify the
 .I original-IP-type-of-service (TOS)
@@ -175,7 +175,7 @@
 as the options when building the original IP header for the injected ICMP 
 unreachable, redirect or time exceeded packet.  IP options can be up to 40 
 bytes in length.  The IP options file must be created manually based upon the 
-desired options.  IP options can also be read from stdin by specifying '-O -' 
+desired options.  IP options can also be read from stdin by specifying '\-O \-' 
 instead of an IP-options-file.
 .IP "-p original-IP-protocol"
 Specify the
@@ -201,8 +201,8 @@
 
 
 within the IP header.  IP fragmentation options can be specified individually 
-or combined into a single argument to the -F command line switch by separating 
-the options with commas (eg. '-FD,M') or spaces (eg. '-FM 223').  The IP 
+or combined into a single argument to the \-F command line switch by separating 
+the options with commas (eg. '\-FD,M') or spaces (eg. '\-FM 223').  The IP 
 fragmentation offset is a 13-bit field with valid values from 0 to 8189.  
 Don't fragment (DF), more fragments (MF) and the reserved flag (RESERVED or RB) 
 are 1-bit fields.
@@ -218,7 +218,7 @@
 as the options when building the IP header for the injected packet.  IP 
 options can be up to 40 bytes in length.  The IP options file must be created
 manually based upon the desired options.  IP options can also be read from 
-stdin by specifying '-O -' instead of an IP-options-file.
+stdin by specifying '\-O \-' instead of an IP\-options\-file.
 .IP "-S source-IP-address"
 Specify the
 .I source-IP-address
